helm_deploy/cla-backend/templates/deployment.yaml

```diff
@@ -104,11 +104,7 @@ spec:
             {{- toYaml .Values.resources | nindent 12 }}
           env:
             - name: POSTGRES_PASSWORD
-              valueFrom:
```

Comment:
I don't understand how removing this doesn't break it for most of our environments. What am I missing?

Reply:
Yes this has broken existing UAT deployments, they would need to rebase from base branch once this goes in.

However it should not break any deployments on staging/production because this only affects deployments that have a local postgres enabled (It's wrapped in `{{- if .Values.localPostgres.enabled -}}`)

Reply:
Also `valueFrom` is only used when referencing secrets.

You can use `Value` when referencing a static(or templated) value.

Reply:
> this only affects deployments that have a local postgres enabled (It's wrapped in {{- if .Values.localPostgres.enabled -}})
Thanks, this was the bit I was missing 👍